src/main/webapp/app/iris/base-chatbot/iris-base-chatbot.component.html

```diff
@@ -0,0 +1,175 @@
+<!-- client -->
+<div class="chat-header">
+    <div class="header-start">
+        <jhi-iris-logo [size]="IrisLogoSize.FLUID" />
+        <div class="word-iris">Iris</div>
+        <a [routerLink]="'/about-iris'" target="_blank">
+            <fa-icon [icon]="faCircleInfo" class="info-button" />
+        </a>
+    </div>
+    <div>
+        @if (rateLimitInfo.rateLimit >= 0) {
+            <div class="toolbox">
+                <span class="rate-limit" [ngbTooltip]="'artemisApp.exerciseChatbot.rateLimitTooltip' | artemisTranslate: { hours: rateLimitInfo.rateLimitTimeframeHours }">
+                    {{ rateLimitInfo.currentMessageCount }}/{{ rateLimitInfo.rateLimit }}
+                </span>
+            </div>
+        }
+        <div class="toolbox">
+            @if (this.messages.length >= 1) {
+                <button id="clear-chat-button" (click)="onClearSession(clearConfirmModal)" class="header-icon">
+                    <fa-icon [icon]="faTrash" />
+                </button>
+            }
+
+            @if (fullSize !== undefined) {
+                <button (click)="fullSizeToggle.next()" class="header-icon">
+                    @if (!fullSize) {
+                        <fa-icon [icon]="faExpand" />
+                    } @else {
+                        <fa-icon [icon]="faCompress" />
+                    }
+                </button>
+            }
+
+            @if (showCloseButton) {
+                <button (click)="closeChat()" class="header-icon">
+                    <fa-icon [icon]="faXmark" />
+                </button>
+            }
+        </div>
+    </div>
+</div>
+<ng-template #clearConfirmModal let-modal>
+    <div class="modal-header">
+        <h4 class="modal-title">
+            <span>{{ 'artemisApp.exerciseChatbot.clearSession.title' | artemisTranslate }}</span>
+        </h4>
+        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss()"></button>
+    </div>
+    <div class="modal-body">
+        <p>{{ 'artemisApp.exerciseChatbot.clearSession.text' | artemisTranslate }}</p>
+    </div>
+    <div class="modal-footer">
+        <button type="button" class="btn btn-danger" (click)="modal.close('confirm')">
+            <span>{{ 'artemisApp.exerciseChatbot.clearSession.submit' | artemisTranslate }}</span>
+        </button>
```

Comment:
We want to use jhiTranslate over artemisTranslate
```suggestion
        <h4 class="modal-title">
            <span jhiTranslate="artemisApp.exerciseChatbot.clearSession.title"></span>
        </h4>
        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss()"></button>
    </div>
    <div class="modal-body">
        <p jhiTranslate="artemisApp.exerciseChatbot.clearSession.text"></p>
    </div>
    <div class="modal-footer">
        <button type="button" class="btn btn-danger" (click)="modal.close('confirm')">
            <span jhiTranslate="artemisApp.exerciseChatbot.clearSession.submit' | artemisTranslate"></span>
        </button>
```